src/files.ts

```diff
@@ -68,6 +93,10 @@ export function installFileMethods(
             else await downloadFile(url, path);
             return path;
         },
+        getIterator() {
```

Comment:
```suggestion
        stream() {
```
needs to be adjusted, as well